src/terms/pairwise.jl

```diff
@@ -0,0 +1,131 @@
+@doc raw"""
+Pairwise terms: Pairwise potential between nuclei, e.g., Van der Waals potentials, such as
+Lennard—Jones terms.
+The potential is dependent on the distance between to atomic positions and the pairwise
+atomic types:
+For a distance `d` between to atoms `A` and `B`, the potential is `V(d, params[(A, B)])`.
+The parameters `max_radius` is of `100` by default, and gives the maximum (reduced) distance
```

Comment:
I'd suggest to put the doc string to the constructor function and not to the struct as the latter is more likely used by people.